### PR TITLE
DB commit

### PR DESCRIPTION
**Pull request by Tolyaspit**
Screenshot 2024-09-24 at 14 26 39
Please check changes and approve it!

### CODE REVIEW

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@Tolyaspit Hi, please fix small comments)

File: src/main/java/NewUserDAOImpl.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    private Connection connection;
+
+    public NewUserDAOImpl(Connection connection) {
+        this.connection = connection;
+    }
+
+    @Override
+    public void saveUser(NewUser user) {
+        String sql = "INSERT INTO User (id, name, email) VALUES (?, ?, ?)";
```
[the remark this answers is not in the document]
Comment (Tolyaspit) [reply src/main/java/NewUserDAOImpl.java:15]: oh , other methods of that classes also?

File: src/main/java/NewTicketDAOImpl.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    private Connection connection;
+
+    public NewTicketDAOImpl(Connection connection) {
+        this.connection = connection;
+    }
+
+    @Override
+    public void saveTicket(NewTicket ticket) {
+        final String sql = "INSERT INTO Ticket (id, ticket_type, start_date, price) VALUES (?, ?, ?, ?)";
```
Comment (Alexmansar) [inline src/main/java/NewTicketDAOImpl.java:15]: image

I mean something like this))

File: src/main/java/NewTicketDAOImpl.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class NewTicketDAOImpl implements NewTicketDAO {
+
```
Comment (Alexmansar) [inline src/main/java/NewTicketDAOImpl.java:6]: and add it here

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@Tolyaspit please, check new comment))

**Review by Alexmansar — APPROVED** [review]
@Tolyaspit Good job)) could be merged)

**Pull request merged**